tests/lmeval/configs/vl_w4a16_actorder_weight.yaml

```diff
@@ -1,19 +1,20 @@
 cadence: "weekly"
-model: Qwen/Qwen2-VL-2B-Instruct
-model_class: TraceableQwen2VLForConditionalGeneration
-recipe: tests/e2e/vLLM/recipes/actorder/recipe_w4a16_actorder_weight.yaml
+model: Qwen/Qwen2.5-VL-7B-Instruct
+model_class: TraceableQwen2_5_VLForConditionalGeneration
+recipe: tests/e2e/vLLM/recipes/actorder/recipe_w4a16_actorder_weight_dampfrac1e-1.yaml
 dataset_id: lmms-lab/flickr30k
-dataset_split: "test[:512]"
+dataset_split: "test[:256]"
 scheme: W4A16_actorder_group
```

Comment:
I can see that this line isn’t touched in this PR, but _should_ the `scheme` be `W4A16_actorder_weight` instead? If so, please update it accordingly to help avoid timings CSV file clashes.

Reply:
Should be resolved now, `scheme` is never used if `recipe` is set, so i got rid of any references when that was the case. PR should be ready for review now. We had to move to a bigger model to have more robust results, so i changed one of the tests to not run GPTQ (it's run in another config). So i did change some file names, do we need to update anything for the timings CSVs?

The 3 tests ran added run in about an hour, maybe 1h15m.

Reply:
Reverted this, please let me know if things still look amiss

Reply:
Looks like `scheme` is removed now, rather than updated, which may cause issues with the naming for the CSV timings during the test. My understanding from skimming the test code is that the model folder and CSV filename that get saved during the test execution is pulled from the config in such a way that:

* If `save_dir` is defined, just use that, otherwise
* Join the `model_name` and `scheme` properties together to create the context-specific folder/filename

```
self.save_dir = eval_config.get("save_dir")
[…]
if not self.save_dir:
    self.save_dir = self.model.split("/")[1] + f"-{self.scheme}"
```

Reply:
This looks good, thanks! I mostly wanted to make sure the `scheme` was correct (based on filename), because we've had collisions in the past related to different configs incorrectly having the same scheme (due to being overlooked, presumably after a copy-paste/starting from duplicated file).